[COMSUP-246] Handling compressed responses in EF (#997)
edgio-docs/edgio-docs: src/guides/v7/edge_functions.md

See the [Edge Function Caching](/guides/edge_functions/caching) guide for more information on caching origin fetch requests.

### Compressed Responses {/* compressed-responses */}

Edge functions do not automatically decompress responses from the origin server, specifically when those responses are encoded with compression methods like `gzip` or `br`. This characteristic becomes relevant if you need to manipulate the response body before sending it to the client, as decompression would be a necessary step in processing the data. Responses that are merely passed through from the origin to the client without modification are not impacted by this behavior.

> To manage this, when making a fetch request within an edge function and intending to manipulate the response body, you should explicitly set request headers to disallow compression. This ensures the response from the origin is not compressed, making it directly accessible for manipulation. The following sample code demonstrates how to modify the request headers:

Reviewer: ```suggestion
If your edge function manipulates the response body of a fetch request, we recommend that you disable compression by setting the `Accept-Encoding` request header. This ensures the response from the origin is not compressed, making it directly accessible for manipulation. The following sample code demonstrates how to disable compression on a fetch request:
```
